platforms/nuget/build-jinja.py

```diff
@@ -0,0 +1,32 @@
+import os
+from jinja2 import Environment, FileSystemLoader
+PATH = os.path.dirname(os.path.abspath(__file__))
+TEMPLATE_ENVIRONMENT = Environment(
+    autoescape=False,
+    loader=FileSystemLoader(os.path.join(PATH, 'templates')),
+    trim_blocks=False)
+
+params = {
+'id': 'OpenCVNuget',
+'version': '4.5.2',
```

Comment:
Version could be automatically parsed from version.hpp in core module.

Reply:
Version now updated to automatically fetch from the version.hpp file.